api/src/main/java/org/openmrs/api/db/hibernate/HibernateProviderDAO.java

```diff
@@ -271,6 +271,12 @@ private <T> T getByUuid(String uuid, Class<T> clazz) {
 		return (T) criteria.uniqueResult();
 	}
 	
+	private <T> T getProviderAttributeTypeByName(String name, Class<T> tClass) {
+		Criteria criteria = getSession().createCriteria(tClass);
+		criteria.add(Restrictions.eq("name", name));
```

Comment:
Is there any reason why you added two methods in this class instead of just one?